src/components/LoanCards/FavoriteStar.vue

```diff
@@ -1,18 +1,50 @@
 <template>
-	<button
-		v-kv-track-event="['Lending', 'click-Favorite star', 'Favorite', loanId, loanId]"
-		@click.prevent.stop="toggleFavorite"
-	>
-		<kv-icon name="star" :from-sprite="true" class="icon-star" :class="{ 'is-favorite': isFavorite }" />
-	</button>
+	<!--
+	*
+	*
+	*
+	*
+	*
+		This is a old file, we don't want to use moving forward
+		if you're looking to implement loan bookmarking behavior
+		please use:
+			src/components/BorrowerProfile/LoanBookmark.vue
+	*
+	*
+	*
+	*
+	*-->
+	<span>
+		<button
+			v-if="!isFavorite"
+			class="tw-inline-flex tw-p-1 tw-cursor-pointer tw-bg-primary tw-rounded tw-m-0.5"
+			@click="toggleFavorite()"
```

Comment:
Please bring the kv-track events back to these buttons

Reply:
Thank you for catching that.